### PR TITLE
v5.30 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft For v5.30 changelog, please help with the following information:

 - Any new config.json settings
 - Database Changes
 - API Changes
 - Websocket Event Changes

**Comment by mkraft** [comment]
> @mkraft For v5.30 changelog, please help with the following information:
> 
> * Any new config.json settings
> * Database Changes
> * API Changes
> * Websocket Event Changes

Release notes:

- Db: upgrade db to 5.30 ([#16453](https://github.com/mattermost/mattermost-server/pull/16453), [@cpanato](https://github.com/cpanato))
- Cleanup config store on server initialization errors ([#16440](https://github.com/mattermost/mattermost-server/pull/16440), [@gabrieljackson](https://github.com/gabrieljackson))
- Added support for processing import files through the API. ([#16062](https://github.com/mattermost/mattermost-server/pull/16062), [@streamer45](https://github.com/streamer45))
- In the event a subscription has a payment and there is no card on the account when that payment is processed, an email will be sent to attempt to get the user to pay up. ([#16442](https://github.com/mattermost/mattermost-server/pull/16442), [@nickmisasi](https://github.com/nickmisasi))
- Pre-package and pre-install the channel export plugin for enterprise-ready builds. ([#16406](https://github.com/mattermost/mattermost-server/pull/16406), [@lieut-data](https://github.com/lieut-data))
- Thread related API routes now include 'teamId' path parameter ([#16304](https://github.com/mattermost/mattermost-server/pull/16304), [@reflog](https://github.com/reflog))

**Review by mkraft — APPROVED** [review]

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,104 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
 
+## Release v5.30
+
+### Compatibility
+  - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
+
+### Highlights
+
+#### Incident Management provided out-of-the-box (E20)
+ - Pre-packaged and pre-installed the channel export plugin for enterprise-ready builds.
```
Comment (itao) [inline source/administration/changelog.md:15]: > Prepackaged and pre-installed the Incident Management as well as Channel Export plugins for enterprise-ready builds.

**Review by itao — CHANGES_REQUESTED** [review]

File: source/administration/changelog.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 ### Highlights
 
 #### Incident Management provided out-of-the-box (E20)
- - Pre-packaged and pre-installed the channel export plugin for enterprise-ready builds.
+ - Prepackaged and pre-installed the Incident Management as well as Channel Export plugins for enterprise-ready builds.
```
Comment (itao) [inline source/administration/changelog.md:15]: Sorry @amyblais it was my mistake, `Pre-packaged` should also have an hyphen. Otherwise LGTM!

**Review by itao — CHANGES_REQUESTED** [review]

**Review by itao — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Command Line Interface (CLI)
+ - Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks.
+ - Added ``mmctl system version`` endpoint to print the remote server version.
+ - Improved the ``mmctl system status`` command output to include all the reported values.
+
+#### Integrations
+ - Updated icon_emoji field in incoming webhooks to allow emojis to be specified with surrounding colons.
+ - Dynamic auto-completion is now supported for built-in slash commands.
+ - Added plugin hooks for ``ReationHasBeenAdded`` and ``ReactionHasBeenRemoved``.
```
Comment (kongr45gpen) [inline source/administration/changelog.md:41]:
```suggestion
 - Added plugin hooks for ``ReactionHasBeenAdded`` and ``ReactionHasBeenRemoved``.
```

(This was a typo in the MR title, see https://github.com/mattermost/mattermost-server/pull/15765/files for the source code)

**Pull request merged**